pkg/rebuild/rebuild/storage.go

```diff
@@ -111,6 +119,9 @@ func DebugStoreFromContext(ctx context.Context) (AssetStore, error) {
 		if strings.HasPrefix(uploadpath, "gs://") {
```

Comment:
same comment as below. maybe use URL parsing here?

Reply:
Used URL parsing for the scheme but it seemed more straightforward to use uploadpath to actually build the AssetStore, using the URL elements would have required some concatenation and would silently drop params.